src/c++/perf_analyzer/main.cc

```diff
@@ -965,6 +999,26 @@ main(int argc, char** argv)
         measurement_request_count = std::atoi(optarg);
         break;
       }
+      case 28: {
+        std::string arg = optarg;
+        if (!arg.empty()) {
+          server_library_path = arg;
+        } else if (kind == cb::TRITON_LOCAL) {
+          std::cerr << "using default server libray path for C API: "
+                    << server_library_path << std::endl;
+        }
+        break;
+      }
+      case 29: {
+        std::string arg = optarg;
```

Comment:
You can just set server_library_path = optarg as is done in the other arg parsing. I don't understand the need for the error message.

Reply:
Needs fix.